deploy/gatekeeper.yaml

```diff
@@ -197,17 +337,18 @@ spec:
           value: gatekeeper-webhook-server-secret
         image: quay.io/open-policy-agent/gatekeeper:v3.0.4-alpha.0
         imagePullPolicy: Always
+        name: manager
```

Comment:
Should we also change `manager` to `gatekeeper-manager` like the clusterrole and clusterrolebinding?

Reply:
Seems redundant given it's in the gatekeeper-controller-manager pod